EIPS/eip-5564.md

```diff
@@ -0,0 +1,162 @@
+---
+eip: 5564
+title: Stealth Address Wallets
+description: Stealth addresses for smart contract wallets
+author: Anton Wahrstätter (@Nerolation)
+discussions-to: https://ethereum-magicians.org/t/eip-5566-stealth-addresses-for-smart-contract-wallets/10614
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-13
+requires: 165
+---
+
+
+## Abstract
+This specification defines a standardized way of creating stealth addresses for smart contract wallets. This EIP enables senders of transactions/transfers to generate private stealth addresses for their recipients that only the recipient can eventually unlock.
+
+## Motivation
+The standardization of stealth address generation may unlock significant privacy potential within Smart Contract Wallets, allowing the recipient of a transfer to remain private when receiving an asset. A Stealth address is generated by the sender from a shared secret between sender and recipient and can only be unlocked by the recipient because only the recipient can compute the matching private key.
+Observers have no possibility of linking the recipient's stealth address to the recipient's identity, leaving only the sender with that information.
```

Comment:
```suggestion
The standardization of stealth address generation may unlock significant privacy potential within Smart Contract Wallets, allowing the recipient of a transfer to remain private when receiving an asset. A Stealth address is generated by the sender from a shared secret between sender and recipient and can only be unlocked by the recipient because only the recipient can compute the matching private key.

Observers have no possibility of linking the recipient's stealth address to the recipient's identity, leaving only the sender with that information.
```